site/content/en/docs/Guides/fleet-updates.md

```diff
@@ -147,6 +147,7 @@ This provides two useful capabilities:
 2. This can also be used to proactively update `GameServer` labels, to effect change in allocation strategy - such as 
    preferring the newer `GameServers` when allocating, but falling back to the older version if there aren't enough 
    of the new ones yet spun up.
+The labels and/or annotations are applied to `GameServers` in a `Fleet` in the order designated by their configured [Fleet scheduling]({{< ref "/docs/Advanced/scheduling-and-autoscaling#fleet-scheduling">}})
```

Comment:
```suggestion

The labels and/or annotations are applied to `GameServers` in a `Fleet` in the order designated by their configured [Fleet scheduling]({{< ref "/docs/Advanced/scheduling-and-autoscaling#fleet-scheduling">}})
```

Just to make it so that the line doesn't end up in the bulleted list above.

![image](https://github.com/googleforgames/agones/assets/298370/3ad32d50-b9b1-4030-9a88-352f25bd15cd)